src/patternfly/components/Dropdown/examples/Dropdown.md

```diff
@@ -149,7 +157,7 @@ The dropdown menu can contain either links or buttons, depending on the expected
 | `.pf-c-dropdown__menu` | `<ul>`, `<div>` | Defines the parent wrapper of the menu items. |
 | `.pf-c-dropdown__menu-item` | `<a>` | Defines a menu item that navigates to another page. |
 | `.pf-c-dropdown__menu-item` | `<button>` | Defines a menu item that performs an action on the current page. |
-| `.pf-c-dropdown__separator` | `<div>` | Defines a separator within the menu. |
+| `.pf-c-dropdown__separator` | `<li>`, `<h4>` | Defines a separator within the menu. |
```

Comment:
maybe I missed this but where is the separator an `h4` ?

Reply:
typo :) should be `<hr>` - updated